spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'user show view', type: :feature do
+  before :each do
+    # visit users_path
+
+    @photo = 'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/microsoft/58/ballot-box-with-x_2612.png'
+    @user2 = User.create!(name: 'Michal Smith', bio: 'Lorem ipsum dolor sit amet', photo: @photo, postsCounter: 3)
+
+    @post1 = @user2.posts.create!(title: 'Business', text: 'lorem_one', commentsCounter: 0, likesCounter: 0)
+    @post2 = @user2.posts.create!(title: 'Education', text: 'lorem_two', commentsCounter: 0, likesCounter: 0)
+    @post3 = @user2.posts.create!(title: 'Science', text: 'lorem_one', commentsCounter: 0, likesCounter: 0)
+
+    visit user_path(@user2)
+  end
+
+  it 'Should see the user\'s profile picture' do
+    image_src = page.find('img')['src']
+    expect(image_src).to eql @photo
+  end
+
+  it 'Should see the user\'s username' do
+    expect(page).to have_content 'Michal Smith'
+  end
+
```

Comment:
- [ ] nice work. please in the show page, kindly add Tests that asserts you can see the number of posts the user has written.